test/parallel/test-webstream-string-tag.js

```diff
@@ -4,15 +4,39 @@ require('../common');
 
 const assert = require('assert');
 
-const classesToBeTested = [ WritableStream, WritableStreamDefaultWriter, WritableStreamDefaultController,
-                            ReadableStream, ReadableStreamBYOBRequest, ReadableStreamDefaultReader,
-                            ReadableStreamBYOBReader, ReadableStreamDefaultController, ReadableByteStreamController,
-                            ByteLengthQueuingStrategy, CountQueuingStrategy, TransformStream,
-                            TransformStreamDefaultController];
+const classesToBeTested = [
+  WritableStream,
+  WritableStreamDefaultWriter,
+  WritableStreamDefaultController,
+  ReadableStream,
+  ReadableStreamBYOBRequest,
+  ReadableStreamDefaultReader,
+  ReadableStreamBYOBReader,
+  ReadableStreamDefaultController,
+  ReadableByteStreamController,
+  ByteLengthQueuingStrategy,
+  CountQueuingStrategy,
+  TransformStream,
+  TransformStreamDefaultController,
+];
 
+/* Commenting forEach Loop as per the document */
```

Comment:
Hey!

You've added a bunch of unrelated changes, could you please revert and keep only the replacement of `forEach` to `for..of`? Also, make sure to follow our [commit guidelines](https://github.com/nodejs/node/blob/main/doc/contributing/pull-requests.md#commit-message-guidelines) otherwise your commit will fail in the CI.

Reply:
Resolved and commited.